docs/en/openmldb_quickstart.md

```diff
@@ -0,0 +1,379 @@
+# OpenMLDB Quick Start
+
+This tutorial provides a quick start guide for OpenMLDB. By establishing a database, importing data, offline feature calculation, SQL solution online, and online real-time feature calculation, the basic usage process of the standalone version of OpenMLDB and the cluster version of OpenMLDB is demonstrated.
```

Comment:
basic steps are: create database, ..., deploy SQL, online ...

Reply:
should we replace all `online/offline calculation` with `online/offline extraction`? @lumianph

Reply:
@lumianph  confirms that extraction is better.